res/img/icons-room-nobg.svg

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<svg
+   xmlns:dc="http://purl.org/dc/elements/1.1/"
+   xmlns:cc="http://creativecommons.org/ns#"
+   xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+   xmlns:svg="http://www.w3.org/2000/svg"
+   xmlns="http://www.w3.org/2000/svg"
+   xml:space="preserve"
+   style="enable-background:new 0 0 25 25;"
+   viewBox="0 0 25 25"
+   y="0px"
+   x="0px"
+   id="Layer_1"
+   version="1.1"><metadata
+   id="metadata11"><rdf:RDF><cc:Work
+       rdf:about=""><dc:format>image/svg+xml</dc:format><dc:type
+         rdf:resource="http://purl.org/dc/dcmitype/StillImage" /><dc:title></dc:title></cc:Work></rdf:RDF></metadata><defs
+   id="defs9">
+	
+	
+
+		
```

Comment:
I'm convinced that whatever thing you're using to do svgs loves to add whitespace

Reply:
Interesting, that must be Inkscape. Could do a final pass at the end when they are final, to shrink them down as much as possible.

Reply:
Interesting, that must be Inkscape. Could do a final pass at the end when they are final, to shrink them down as much as possible.